src/AppContext.js

```diff
@@ -89,9 +82,13 @@ export class AppProvider extends Component {
             poster,
           },
         ],
-        message: messageText,
+        message: messages.ADD_MESSAGE,
       }));
     else if (type === actionType.REMOVE) {
+      let messageText = messages.REMOVE_MESSAGE;
```

Comment:
a nie możesz od razu? w miejscu wykorzystania?
```js
...
message: id !== this.state.movieID ? "" : messages.REMOVE_MESSAGE;
```
to Twoje mutowanie zmiennej jest brzydką praktyką